### ISSUE TITLE
Discussion: Reduce robocoop noise

### ISSUE THREAD

**Issue by alexander-bobin** [slwen/robocoop#8]
It seems that about half the messages in the channel are from robocoop. What can be done to reduce that noise and in doing so, make robocoop's messages more valuable to users.

Ideas:
- Maybe instead of replying "Thank you ... " after "@ robocoop: I did ...", robocoop could add a robocoop reaction to the message to show that it has been recorded

**Comment by slwen** [comment]
I agree, the main idea behind the current thank you message is to confirm the users action. I totally like the idea of using reactions instead. 👍
